src/pages/[platform]/build-a-backend/storage/list-files/index.mdx

```diff
@@ -104,28 +108,14 @@ function processStorageList(response) {
 
 This places each item's data inside a special `__data` key.
 
-### List files without pagination
-
-You can set the `listAll` option to true if you want to get all your files that match the prefix without paginating them.
-
-```javascript
-import { list } from 'aws-amplify/storage';
-
-try {
-  const response = await list({
-    path: 'photos/'
-    // Alternatively, path: ({identityId}) => `album/{identityId}/photos/`
-    options: {
-      listAll: true
-    }
-  });
-  // render list items from response.items
-} catch (error) {
-  console.log('Error ', error);
-}
-```
+### More `list` options
```

Comment:
This is an anti-pattern. It's kinda fo the same as "advanced workflows" in the end it doesn't really mean anything. The table below can have their won headings:

"### List app files under the specified path"

"### Paginate through files under a specified path"

"### Use with S3 Transfer Acceleration"